src/toolbars/CutCopyPasteToolBar.cpp

```diff
@@ -0,0 +1,174 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  CutCopyPasteToolBar.cpp
+
+  ksoze95
+
+  See CutCopyPasteToolBar.h for details
+
+*******************************************************************//*!
+
+\class CutCopyPasteToolBar
+\brief A ToolBar that has the cut, copy and paste buttons on it.
+
+  This class, which is a child of Toolbar, creates the
+  window containing interfaces to cut/copy/paste
+  functions that are otherwise only available through
+  menus. The window can be embedded within a normal project
+  window, or within a ToolBarFrame.
+
+  All of the controls in this window were custom-written for
+  Audacity - they are not native controls on any platform -
+  however, it is intended that the images could be easily
+  replaced to allow "skinning" or just customization to
+  match the look and feel of each platform.
+
+*//*******************************************************************/
```

Comment:
This entire file is a cut-and-paste of EditToolBar with some modifications.

These two beg to be unified into a common base class, simply parameterized by a table of buttons and commands.

Such a simple, table driven toolbar might even become a useful toolkit neutral facade if it is done right.

None of this comment is a request for changes now.